src/main/java/de/techdev/portal/domain/trackr/TrackrConfiguration.java

```diff
@@ -12,6 +11,12 @@
 @Configuration
 class TrackrConfiguration {
 
+    @Value("${trackr.oauth.resourceId}")
+    private String trackrResourceId = "techdev-services";
```

Comment:
Please don not initialize this, since it is not obvious if the default or the property value will be set in the end.